src/main/scala/com/goyeau/kubernetes/client/KubernetesClient.scala

```diff
@@ -26,10 +26,10 @@ case class KubernetesClient[F[_]: ConcurrentEffect](httpClient: Client[F], confi
 }
 
 object KubernetesClient {
-  def apply[F[_]: ConcurrentEffect](config: KubeConfig): Resource[F, KubernetesClient[F]] =
+  def apply[F[_]: ConcurrentEffect](config: KubeConfig)(implicit cs: ContextShift[IO]): Resource[F, KubernetesClient[F]] =
```

Comment:
```suggestion
  def apply[F[_]: ConcurrentEffect: ContextShift](config: KubeConfig): Resource[F, KubernetesClient[F]] =
```
With this we are not forcing the IO monad

Reply:
yes, I tried generic `F` as effect-type first, but Cats-effect requires:
```
def fromFuture[A](iof: IO[Future[A]])(implicit cs: ContextShift[IO])`
```
we have to pass ContextShift[IO] for IO explicitly. I can try to replace it with https://typelevel.org/cats-effect/typeclasses/async.html#async

Reply:
changed to `Async`